packages/in_app_purchase/in_app_purchase_ios/test/in_app_purchase_ios_platform_test.dart

```diff
@@ -273,6 +273,60 @@ void main() {
       expect(completerError.message, 'ios_domain');
       expect(completerError.details, {'message': 'an error message'});
     });
+    test(
```

Comment:
nit: could use an empty line to make it clear a new test starts here:
```suggestion

    test(
```